### ISSUE TITLE
[Bug]: glue catalog exporter is broken for writes done from glue studio

### ISSUE THREAD

**Issue by talSofer** [treeverse/lakeFS#7023]
### What happened?

Hive tables are not exported into Glue data catalog while using AWS glue studio to write to lakefs.  
        
Steps to Reproduce:
1. Follow https://docs.lakefs.io/integrations/glue_metastore.html#configure-action-hooks to export hive table from lakefs into glue database. Use a Spark script in AWS Glue studio to [write data to the table](https://docs.lakefs.io/integrations/glue_metastore.html#write-some-table-data) as parquet. 
2. use `df.write.partitionBy(“type”,“weight”).mode(“overwrite”).parquet(dataPath)` to insert data into the table. 
3. complete the exporter guide and check `{storage_namespace}/_lakefs/exporter`, which is the object storage location where symlinks are saved - this storage space is not created. 

### Expected behavior

Table should be accessible for reads in Glue data catalog. 

### lakeFS version

1.2.0

### How lakeFS is installed

cloud

**Comment by Isan-Rivkin** [comment]
Additional Context: 
when specifying  `partitionBy("type", "weight")`, Spark will create a directory structure based on the values in the specified columns (“type” and “weight” in your case). That empty dir created is actually translated to an empty file in lakeFS, not a dir ofcourse since there’s no such thing as dirs so very confusing.
In lakeFS when iterating the “partitions” the no-name empty file comes first (alphabetic order) and it seems like a partition with no files inside, hence it get's skipped and moves to the next partition. 
it should work after removing the empty path’s.

The lua function [extract_partition_pager](https://github.com/treeverse/lakeFS/blob/master/pkg/actions/lua/lakefs/catalogexport/hive.lua#L34) is the one returning that there are no more objects.

**Comment by itaiad200** [comment]
> it should work after removing the empty path’s.

I think that's a short term option to unblock @talSofer. If empty files are expected during writes to mark empty partitions, then we should be able to support that as well.

**Comment by talSofer** [comment]
> f empty files are expected during writes to mark empty partitions, then we should be able to support that as well.

I agree. Even if this only happens while writing from glue studio or other AWS tools - this exporter is about accessing data from glue data catalog, then it should work for writes done by tools of the same ecosystem.
